ci/travis/ci.sh

```diff
@@ -447,9 +447,10 @@ init() {
 }
 
 build() {
+  # NOTE: Do not add build flags here. Use .bazelrc and --config instead.
+  bazel build -k "//:*"  # Full build first, since pip install will build only a subset of targets
+
```

Comment:
why?

Reply:
To make sure targets other than `ray_pkg` keep building (at least on Windows).